Tools/tsan/suppressions_free_threading.txt

```diff
@@ -0,0 +1,51 @@
+# This file contains suppressions for the free-threaded build. It contains the
+# suppressions for the default build and additional suppressions needed only in
+# the free-threaded build.
+#
+# reference: https://github.com/google/sanitizers/wiki/ThreadSanitizerSuppressions
+
+## Default build suppresssions
+
+race:get_allocator_unlocked
+race:set_allocator_unlocked
+race:mi_heap_visit_pages
+race:_mi_heap_delayed_free_partial
+
+## Free-threaded suppressions
+
+race:_add_to_weak_set
```

Comment:
Not yet, working on it :)

Reply:
I'd like to start with a simple list in an omnibus issue like https://github.com/python/cpython/issues/117657 so that there's a single place that people can go look to track who is working on what races.

I expect many of the races to have simple fixes, but we can break off the more complex races into their own issues as needed.

The work of collecting stacktraces and repros is work that can be split up once this PR is landed and we have a way of coordinating the work.

Reply:
Sounds good. I threw together a quick script to massage the TSAN output from the tests into an [index](https://mpage.github.io/cpython_tsan_races/) that we can use to look for stacktraces and tests that should reproduce the races.